src/babel/__tests__/unit.test.js

```diff
@@ -753,14 +842,14 @@ describe('babel plugin', () => {
       "import { di as _di } from "react-magnetic-di";
       import Modal, { config } from 'modal';
       function createClass() {
-        var _class;
+        var _MyModal;
```

Comment:
Seems like this is making the pipeline fail

Reply:
That is strange change, because this is what it _wants_ to be.